.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,64 @@
+name: integration-tests
+on:
```

Comment:
You should allow manual start of this workflow: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

Reply:
Additionally, I suggest to run this workflow when PR is approved: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review